introspection-engine/connectors/sql-introspection-connector/src/misc_helpers.rs

```diff
@@ -18,46 +18,46 @@ pub fn is_migration_table(table: &Table) -> bool {
 }
 
 pub(crate) fn is_prisma_1_point_1_join_table(table: &Table) -> bool {
-    table.columns.len() == 2
-        && table.foreign_keys.len() == 2
-        && table.foreign_keys[0].referenced_table < table.foreign_keys[1].referenced_table
-        && table.name.starts_with("_")
-        && table
-            .columns
-            .iter()
-            .find(|column| column.name.to_lowercase() == "a")
-            .is_some()
-        && table
-            .columns
-            .iter()
-            .find(|column| column.name.to_lowercase() == "b")
-            .is_some()
-        && table.indices.len() >= 1
-        && table.indices.last().unwrap().columns.len() == 2
-        && table.indices.last().unwrap().tpe == IndexType::Unique
+    table.columns.len() == 2 && table.indices.len() >= 2 && shared(table)
```

Comment:
I wanted to leave this open in case people have sth like this:
Index(A)
Index(B)
Primary(A,B)
I am not aware of us doing it like this, but I wanted to leave that flexibility.

Reply:
yup sounds reasonable